prefect_snowflake/database.py

```diff
@@ -381,6 +391,7 @@ async def fetch_all(
         self,
         operation: str,
         parameters: Optional[Dict[str, Any]] = None,
+        cursor_class: Optional[type[SnowflakeCursor]] = SnowflakeCursor,
```

Comment:
I saw the comment about `cursor_type`, but also these fields should be typed as `type[SnowflakeCursor]` since I don't think that `None` is a valid value for this argument.
```suggestion
        cursor_type: type[SnowflakeCursor] = SnowflakeCursor,
```

Reply:
@desertaxle Sorry missed this and made another commit. I added this in.